[DOC+][Troubleshooting] Kibana Server not Ready yet (#158379)
elastic/kibana: docs/setup/access.asciidoc

[[not-ready]]
=== {kib} not ready

If you receive a {kib} UI error on any page that the `{kib} Server is not Ready yet`, sequentially:

. Confirm connectivity to {es} from within a {kib} node:
+
[source,sh]
----
`curl -XGET elasticsearch_ip_or_hostname:9200/`
----

> . Guarantee the health of Kibana-backing indices which should all three report and should each show `status:green` and `status:open`:

Reviewer: ```suggestion
. Guarantee the health of the three {kib}-backing indices.  All indices must appear and display `status:green` and `status:open`:
```
